src/App.jsx

```diff
@@ -0,0 +1,14 @@
+import { Routes, Route } from 'react-router-dom';
+import ItemPage from './pages/items';
+import AddItem from './pages/add-item';
+import IndexPage from './pages/indexPage';
+
+export default function App() {
+  return (
+    <Routes>
+      <Route path='/' element={<IndexPage />} />
```

Comment:
💊 제안
index 속성을 이용해 시작페이지라는 것을 더 명확하게 알 수 있게 하시는 것을 추천드립니다~

```suggestion
      <Route index element={<IndexPage />} />
```